libraries/node-core-library/src/Resolve.ts

```diff
@@ -3,15 +3,23 @@
 
 import { sync } from 'resolve';
 import * as path from 'path';
-import { FileSystem, PackageJsonLookup } from '@rushstack/node-core-library';
+
+import { PackageJsonLookup } from './PackageJsonLookup';
+import { FileSystem } from './FileSystem';
 
 const packageJsonLookup: PackageJsonLookup = new PackageJsonLookup();
 
+/**
+ * @alpha
+ */
 export interface IResolvePackageOptions {
   doNotResolveSymlinks: boolean;
 }
 
-export class ResolveUtilities {
+/**
+ * @alpha
```

Comment:
Let's move this functionality into the `Import` API.  What we are resolving are NPM package imports, so they seem logically related enough to be the same API.